docset/winserver2022-ps/appx/Remove-AppSharedPackageContainer.md

```diff
@@ -0,0 +1,90 @@
+﻿---
+external help file: Microsoft.Windows.Appx.PackageManager.Commands.dll-Help.xml
+Module Name: appx
+online version: https://docs.microsoft.com/powershell/module/appx/move-appxpackage?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+---
+
+# Remove-AppSharedPackageContainer
+
+## SYNOPSIS
+Removes the shared package container.
+
+## SYNTAX
+
+```
+Remove-AppSharedPackageContainer [-Name] <String> [-ForceApplicationShutdown] [-AllUsers] [<CommonParameters>]
+```
+
+## DESCRIPTION 
+The Remove-AppSharedPackageContainer cmdlet removes the shared package container definiton for the particular user.
```

Comment:
Cmdlet names should be in backticks in a paragraph. But you don't need the cmdlet name here because it is redundant.
```suggestion
The cmdlet removes the shared package container definition for the particular user.
```